piecedirectory/doctor.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/filecoin-project/lotus/chain/types"
 	"github.com/ipfs/go-cid"
 	logging "github.com/ipfs/go-log/v2"
+	verifregtypes "github.com/filecoin-project/go-state-types/builtin/v9/verifreg"
```

Comment:
Can you please check with v12 verifreg? We are trying to use latest actors as much as possible.

Reply:
Are you sure that will work? The v9 one is the one used in the return type of the API call

Reply:
Ahhh! Looks like a Lotus side dependency. Merging.